Makefile

```diff
@@ -774,6 +774,9 @@ test-ci: all
 	@PONYPATH=. $(PONY_BUILD_DIR)/ponyc -d -s examples
 	@./examples1
 	@rm examples1
+	@$(PONY_BUILD_DIR)/ponyc --antlr > pony.g.new
+	@diff pony.g pony.g.new
```

Comment:
It will fail. `diff` returns 0 if the files are equal and 1 if they are different.